weave/weave_client.py

```diff
@@ -310,7 +317,7 @@ def make_client_call(
         id=server_call.id,
         inputs=from_json(server_call.inputs, server_call.project_id, server),
         output=output,
-        summary=server_call.summary,
+        summary=dict(server_call.summary if server_call.summary else {}),
```

Comment:
can you explain why this change is needed? Seems to effect some tests and I am curious what was the need for the here (perhaps left over from pydantic?)

Reply:
I can modify this ternary so we dont return {} and mess up tests, but this is necessary because we return SummaryMap, but i left call.summary as a dict, because the rollup of arbritrary keys got a bit complex using typeddicts